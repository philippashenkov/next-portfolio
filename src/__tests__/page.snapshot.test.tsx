import { render } from "@testing-library/react";
import React from "react";
import Home from "../app/page";

// Snapshot test for the page structure (client-rendered)

describe("Home page", () => {
  it("matches snapshot", () => {
    const { asFragment } = render(<Home />);
    expect(asFragment()).toMatchSnapshot();
  });
});
