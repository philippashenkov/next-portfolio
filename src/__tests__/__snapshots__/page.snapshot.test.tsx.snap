// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Home page > matches snapshot 1`] = `
<DocumentFragment>
  <main
    class="page hero"
  >
    <div
      class="hero__scene"
    >
      <div
        style="position: relative; width: 100%; height: 100%;"
      >
        <div
          style="position: absolute; inset: 0; pointer-events: none; z-index: 2;"
        />
      </div>
    </div>
    <section
      class="hero__content"
    >
      <div
        class="retro-card hero__card"
        style="padding: 1.75rem 2rem; width: 100%;"
      >
        <h1
          style="font-family: var(--font-orbitron); font-size: 36px; line-height: 1.2;"
        >
          Hi, I’m Philipp
        </h1>
        <div
          style="margin-top: 8px; opacity: 0.95;"
        >
          Frontend & Creative Developer
        </div>
        <div
          class="hero-type"
          style="position: relative;"
        >
          <style>
            
        @keyframes tw-blink { 0%, 45% { opacity: 1 } 50%, 100% { opacity: 0 } }
        .tw-cursor { display: inline-block; margin-left: 2px; animation: tw-blink 1s steps(2, start) infinite; }
      
          </style>
          <div
            class="hero-type-line"
            style="white-space: pre-wrap;"
          >
            <span
              aria-hidden="true"
              class="tw-cursor"
            >
              ▋
            </span>
          </div>
          <div
            class="hero-type-line"
            style="white-space: pre-wrap;"
          />
          <div
            class="hero-type-line"
            style="white-space: pre-wrap;"
          />
          <div
            class="hero-type-line"
            style="white-space: pre-wrap;"
          />
          <div
            class="hero-type-line"
            style="white-space: pre-wrap;"
          />
          <div
            class="hero-type-line"
            style="white-space: pre-wrap;"
          />
          <div
            class="hero-type-line"
            style="white-space: pre-wrap;"
          />
          <div
            class="hero-type-line"
            style="white-space: pre-wrap;"
          />
          <div
            class="hero-type-line"
            style="white-space: pre-wrap;"
          />
          <div
            class="hero-type-line"
            style="white-space: pre-wrap;"
          />
          <div
            class="hero-type-line"
            style="white-space: pre-wrap;"
          />
          <div
            class="hero-type-line"
            style="white-space: pre-wrap;"
          />
          <div
            class="hero-type-line"
            style="white-space: pre-wrap;"
          />
          <div
            class="hero-type-line"
            style="white-space: pre-wrap;"
          />
          <div
            class="hero-type-line"
            style="white-space: pre-wrap;"
          />
          <div
            class="hero-type-line"
            style="white-space: pre-wrap;"
          />
          <div
            class="hero-type-line"
            style="white-space: pre-wrap;"
          />
          <div
            class="hero-type-line"
            style="white-space: pre-wrap;"
          />
          <div
            class="hero-type-line"
            style="white-space: pre-wrap;"
          />
          <div
            class="hero-type-line"
            style="white-space: pre-wrap;"
          />
          <div
            class="hero-type-line"
            style="white-space: pre-wrap;"
          />
        </div>
      </div>
    </section>
  </main>
</DocumentFragment>
`;
